nengo/simulator.py

```diff
@@ -55,6 +55,12 @@ def __len__(self):
 class Simulator(object):
     """Reference simulator for Nengo models."""
 
+    unsupported = []
+    """Defines features unsupported by a simulator. The format is a list of
```

Comment:
I'm not sure if we want this in the simulator docstring, because it's for developers not users, and I think we should keep the docstrings targeted at users. It would also be good to give some examples of how it's used in practice; we could promote Nengo OCL as one of these examples once that stuff is merged (which won't happen until Nengo 2.1.0 is released). So maybe just turn this into a comment for now, and we can worry about better documentation down the road? I could even see a "backend developer" document of some sort that takes developers through all the utilities and things they should know for developing a new backend.